src/components/character-controller.js

```diff
@@ -137,6 +140,10 @@ AFRAME.registerComponent("character-controller", {
       if (userinputAngularVelocity !== null && userinputAngularVelocity !== undefined) {
         this.angularVelocity = userinputAngularVelocity;
       }
+      const dCharSpeed = userinput.get(paths.actions.dCharSpeed) || 0;
+      if (enableWheelSpeed && dCharSpeed) {
```

Comment:
Minor but I would move the check for the `enableWheelSpeed` to surround this whole new block. Then I am not sure the aCharSpeed check is needed, as it would be handled by the bounds check.